### PR TITLE
Rainbows! Fixes #21

### LINKED ISSUE #21: [major] no rainbow
kicl needs a rainbow method in the format / color class

### PR DESCRIPTION
**Pull request by ammaraskar**
mmyes

### CODE REVIEW

**Comment by dequis** [comment]
:+1:

File: src/main/java/org/kitteh/irc/client/library/IRCFormat.java [hunk 1]
```diff
@@ -122,4 +122,22 @@ public String withBackground(IRCFormat background) {
...
+
+    private static final IRCFormat[] RAINBOW = {RED, BROWN, OLIVE, YELLOW, DARK_GREEN, GREEN, TEAL, BLUE, MAGENTA, PURPLE};
+    /**
+     * Adds rainbow colors to every character of a message
+     *
+     * @param message The message to color
+     * @return The colorized message
+     */
+    public static String rainbowize(String message) {
+        Sanity.nullCheck(message, "Can't rainbowize a null message :(");
```
Comment (dequis) [inline src/main/java/org/kitteh/irc/client/library/IRCFormat.java]: This might be too sad

**Comment by kashike** [comment]
:+1:

**Comment by jamierocks** [comment]
Looks better than my attempt -- https://github.com/KittehOrg/KittehIRCClientLib/pull/16 :P

**Comment by mbax** [comment]
https://github.com/KittehOrg/KittehIRCClientLib/commit/6bfa0f41bc4aa59d134a30075e8899597ad4a431

image

**Pull request closed without merging**